.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,24 @@
+name: continuous-integration
+on: [push, pull_request]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7, 3.8, 3.9]
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+    - name: Setup python
+      uses: actions/setup-python@v1
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dev dependencies
+      run: |
+        pip install --upgrade pip wheel
+        pip install -e .[dev,test]
+    - name: Test with pytest
+      run: |
+          pytest tests -r a --cov=fair --cov-report=xml
+    - name: Coverage report
+      run: codecov
```

Comment:
Could use the below if you want to upload the coverage report

```
- name: Upload coverage to Codecov
      if: startsWith(runner.os, 'Linux') && matrix.python-version == 3.7
      uses: codecov/codecov-action@v1
      with:
        file: ./coverage.xml
```

Reply:
Does this replace the `run: codecov` line or come after?

Reply:
replace (but maybe yours already works, sort of unclear to me....)